src/imgui/ImGuiManager.hh

```diff
@@ -190,7 +197,13 @@ private:
 		PersistentElement{"mainMenuBarUndocked", &ImGuiManager::mainMenuBarUndocked},
 		PersistentElement{"mainMenuBarFade",     &ImGuiManager::menuFade},
 		PersistentElement{"windowPos",           &ImGuiManager::windowPos},
-		PersistentElement{"statusBarVisible",    &ImGuiManager::statusBarVisible}
+		PersistentElement{"statusBarVisible",    &ImGuiManager::statusBarVisible},
+		PersistentElement{"statusbarItemvisibility.fps",                 &ImGuiManager::statusBarItemVisibilityFps},
```

Comment:
This line has inconsistent capitalization.

Reply:
Will fix.